tests/scripts/thread-cert/Cert_5_3_02_RealmLocal.py

```diff
@@ -96,14 +96,14 @@ def test(self):
                 self.assertTrue(self.nodes[LEADER].ping(addr, size=256))
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1', size=256))
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1'))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2, size=256))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2))
```

Comment:
Thread 1.1 specifies that IPv6 messages destined to the Realm-Local All Nodes address `ff03::1` are not delivered to SEDs.  However, Realm-Local All **Thread** Nodes address (the last two pings in this test) are delivered to SEDs.